BackendWebApp/src/main/java/uk/ac/ebi/biostd/webapp/application/configuration/ConfigProperties.java

```diff
@@ -37,9 +38,9 @@ public String get(String property) {
     }
 
     private void loadProperties(ServletContext context) throws IOException {
-        String config = getConfigFileLocation();
+        File config = Paths.get(getConfigFileLocation()).toFile();
 
-        if (StringUtils.isNotBlank(config)) {
+        if (config.exists()) {
```

Comment:
@jhoanmanuelms bug was file url exist but config file not, so base dir property was not loaded correctly.